_posts/ja/posts/2020-08-01-release-0.20.1.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Bitcoin Core 0.20.1 リリース
+name: blog-release-0.20.1
+id: ja-blog-release-0.20.1
+lang: ja
+permalink: /ja/2020/08/01/release-0.20.1/
+type: posts
+layout: post
+
+## If this is a new post, reset this counter to 1.
+version: 1
+
+## Only true if release announcement or security annoucement.  English posts only
+announcement: 1
```

Comment:
This should be `0` or commented out.

Reply:
@harding thanks your review. I remove it.